src/pages/Host/HostStair/HostMap.js

```diff
@@ -0,0 +1,86 @@
+import React, { useRef, useState } from 'react';
+import { Map, MapMarker } from 'react-kakao-maps-sdk';
+import styled from 'styled-components';
+import variables from '../../../styles/variables';
+import HostResult from '../HostResult';
+
+const HostMap = () => {
+  const mapRef = useRef();
+  const [lat, setLat] = useState(37.5063);
+  const [lng, setLng] = useState(127.0536);
+
+  return (
+    <S.HostMapBox>
+      <Map // 지도를 표시할 Container
+        center={{ lat: 37.5063, lng: 127.0536 }}
+        style={{
+          // 지도의 크기
+          width: '99.8%',
+          height: '100%',
+        }}
+        level={3} // 지도의 확대 레벨
+        ref={mapRef}
+      >
+        <S.LocationButton
+          onClick={() => {
+            const map = mapRef.current;
+            setLng(map.getCenter().getLng());
+            setLat(map.getCenter().getLat());
+          }}
+        >
+          이 주소가 정확하나요?
+        </S.LocationButton>
+        <MapMarker position={{ lat: lat, lng: lng }}></MapMarker>
```

Comment:
key값과 value에 쓰는 변수명이 같으면 축약형으로 사용해도 좋습니다. 
```suggestion
        <MapMarker position={{ lat , lng }}></MapMarker>
```